### ISSUE TITLE
MongoDB SRV record not working

### ISSUE THREAD

**Issue by TheDanielMoli** [oven-sh/bun#8758]
### What version of Bun is running?

1.0.26+c75e768a6

### What platform is your computer?

Darwin 23.3.0 x86_64 i386

### What steps can reproduce the bug?

Use the Node.js default MongoDB client (MongoClient) to connect to a replica set cluster with an SRV record. Aka, URI should look like this: `mongodb+srv://...`

### What is the expected behavior?

Connecting to MongoDB.

### What do you see instead?

```
err: {
    "type": "Error",
    "message": "txt lookup failed: No data record of requested type",
    "stack":
          
    "code": "DNS_ENODATA"
}
```

### Additional information

It works properly with Node.js 18 and Node.js 20.

**Comment by Electroid** [comment]
Duplicate of #5498

**Comment by rnldsalili** [comment]
This issue was fixed on Bun v1.1.7
